From 43ad0073f2ff25b732413b9f15d67cb135d0583d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marco=20Trevisan=20=28Trevi=C3=B1o=29?= <mail@3v1n0.net>
Date: Mon, 8 Jul 2019 17:12:25 +0200
Subject: [PATCH] Makefile-tests: Fix paths for stacking metatests

In commit 29ffaa6f8 and commit 8abd1b009 were added stacking tests with wrong
paths.

Fix them.

Realated to: https://gitlab.gnome.org/GNOME/mutter/merge_requests/307

# Conflicts:
#	src/Makefile-tests.am

(cherry picked from commit 4349e682f8d2c37a27c1dab5f636f8bcbbae0a23)
---
 src/Makefile-tests.am | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/Makefile-tests.am b/src/Makefile-tests.am
index 7cfae2371..7c1dd6040 100644
--- a/src/Makefile-tests.am
+++ b/src/Makefile-tests.am
@@ -15,10 +15,10 @@ dist_stacking_DATA =						\
 	$(srcdir)/tests/stacking/basic-x11.metatest 	\
 	$(srcdir)/tests/stacking/basic-wayland.metatest	\
 	$(srcdir)/tests/stacking/closed-transient.metatest	\
-	$(srcdir)/closed-transient-no-input-no-take-focus-parent.metatest	\
-	$(srcdir)/closed-transient-no-input-no-take-focus-parents.metatest	\
-	$(srcdir)/closed-transient-no-input-parent.metatest	\
-	$(srcdir)/stacking/closed-transient-no-input-parent-delayed-focus-default-cancelled.metatest	\
+	$(srcdir)/tests/stacking/closed-transient-no-input-no-take-focus-parent.metatest	\
+	$(srcdir)/tests/stacking/closed-transient-no-input-no-take-focus-parents.metatest	\
+	$(srcdir)/tests/stacking/closed-transient-no-input-parent.metatest	\
+	$(srcdir)/tests/stacking/closed-transient-no-input-parent-delayed-focus-default-cancelled.metatest	\
 	$(srcdir)/tests/stacking/minimized.metatest 	\
 	$(srcdir)/tests/stacking/mixed-windows.metatest     \
 	$(srcdir)/tests/stacking/set-parent.metatest	\
-- 
2.20.1

